UnityProject/Assets/Scripts/Objects/Nuke.cs

```diff
@@ -284,7 +284,8 @@ IEnumerator TickTimer()
 		while (CurrentTimerSeconds > 0)
 		{
 			CurrentTimerSeconds -= 1;
-			SoundManager.PlayNetworkedAtPos("TimerTick", gameObject.AssumedWorldPosServer());
+			// JESTER
+			//SoundManager.PlayNetworkedAtPos("TimerTick", gameObject.AssumedWorldPosServer());
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)